git/dogfood.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+featureBranch=$(git rev-parse --abbrev-ref HEAD)
+dogfoodBranch=dogfood/$featureBranch
+
+info() {
+    echo "* $@"
+}
+
+branchExists() {
+    git rev-parse -q --verify "$1" >/dev/null
+}
+
+createDogfoodBranch() {
+    git branch "$dogfoodBranch"
+}
+
+rebaseDogfoodBranch() {
+    git checkout "$dogfoodBranch"
+    git rebase "$featureBranch"
+    git checkout -
+}
+
+if [ "$featureBranch" = "master" ]; then
```

Comment:
could you please also add a condition on release branches "branch-*" ? Thanks

Reply:
Added.